style.css

```diff
@@ -0,0 +1,193 @@
+:root {
+    --black: #000000;
+    --white: #ffffff;
+    --light-orange: #9f5757;
+    --blue-black: #273852;
+    --light-gray: #aaaaaa;
+    --orange: #d35b44;
+}
+
+html, body{
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    height: 100vh;
+    font-family: 'Roboto', sans-serif;
+}
+
+.d-flex{
+    display: flex;
+}
+
+.row{
+    flex-direction: row;
+}
+
+.col{
+    flex-direction: column;
+}
+
+.center{
+    justify-content: center;
+    align-items: center;
+}
+
+header{
+   background-color: var(--blue-black);
+   width: 100%;
+}
+
+header > div{
+    text-transform: uppercase;
+    margin: 20px;
+    background-color: var(--white);
+    font-weight: bold;
+    border: 2px solid var(--light-orange);
+    padding: 12px;
+}
+
+#title{
+    font-size: 2rem;
+    margin: 0px 20px 2px 20px;
+}
+
+#sub-title{
+    font-size: small;
+}
+
+main{
+    flex: 1 0 auto;
+    margin: 24px 28px;
+}
+
+#articles-header{
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+    border-bottom: 1.5px solid var(--black);
+}
+
+#category > label, #subscribe-form > label{
+    text-transform: uppercase;
+    font-size: small;
+    font-weight: bolder;
+    margin-bottom: 8px;
+}
+
+#filter, #text-subscribe{
+    text-transform: capitalize;
+    padding: 4px 12px;
+    height: 34px;
+    width: 14rem;
+    border: 1px solid var(--black);
+}
+
+#main-title{
+    font-size: 1.6rem;
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+#text-subscribe{
+    height: 24px !important;
+}
+
+#btn-subscribe{
+    height: 2.12rem;
+    background-color: var(--black);
+    color: var(--white);
+    border: var(--black);
+    cursor: pointer;
+}
+
+.article-container{
+    width: 70%;
```

Comment:
just check if we can use fixed-size padding instead of width in percentage, I think it would be difficult to fit the design in all screens and we would end up writing media queries.

Reply:
Actually for desktop screen it was having width 70% and for tab and mobile screen 100%, so that's why I did it.